flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaOptions.java

```diff
@@ -222,8 +226,17 @@ private static void validateSinkSemantic(ReadableConfig tableOptions) {
 	// Utilities
 	// --------------------------------------------------------------------------------------------
 
-	public static String transformSemantic(String semantic){
-		return semantic.toUpperCase().replace('-', '_');
+	public static KafkaSemantic getSinkSemantic(String semantic){
```

Comment:
`SinkSemantic` or `KafkaSinkSemantic`? It maybe confusing that it also works for source.